test/test_framework.py

```diff
@@ -1369,3 +1377,144 @@ class StatefulObject(Object):
         self.assertEqual(parent._stored.bar, 4)
         # TODO: jam 2020-01-30 is there a clean way to tell that
         #       parent._stored._data.dirty is False?
+
+
+@patch('sys.stderr', new_callable=io.StringIO)
+class BreakpointTests(unittest.TestCase):
+
+    def setUp(self):
+        tmpdir = Path(tempfile.mkdtemp())
+        self.addCleanup(shutil.rmtree, str(tmpdir))
+
+        self.framework = Framework(tmpdir / "framework.data", tmpdir, None, None)
+        self.addCleanup(self.framework.close)
+
+    def test_ignored(self, fake_stderr):
+        # it doesn't do anything really unless proper environment is there
```

Comment:
I'd encourage using proper sentences for comments, starting with capitals, and including punctuation, etc. Unless it's a continuation of an earlier phrase of course. It makes it easier to read because that's how we normally do it everywhere else, and also plays a trick on your head when writing the comment, which is reminding you that the reader (potentially yourslef in a few months) is out of context and might need an extra hint.

Also, I believe that kind of comment, that explains what a test does, is typically done in Python by using a docstring for the method. That's different from the kind of comment that talks about what the next line or two does, of course.

These should be global conventions with the project, so we need to sync everyone with them.

Reply:
Improved the comments, thanks.

I stopped using docstring in test methods as some runner used them in the result output, and was a mess. Don't remember which one was... nosetest?

Anyway, we should define what we want to do as a team in this regard, which is tied to which runner we want to end up using.